VS 21 February/VS 21 February/Program.cs

```diff
@@ -1,102 +1,146 @@
 ﻿using System;
-
 class Hello
 {
     static void Main(string[] args)
     {
-        //First Home Task
-        VS_21_February.Phone iphone_13 = new VS_21_February.Phone();
+        ////HomeTask 1 lesson 6
+        VS_21_February.Triangles triangle_1 = new VS_21_February.TriangleBuilder().CreateTriangle(5, 5, 8);
```

Comment:
Use namespace instead of VS_21_February.Triangles

Reply:
fixed